libs/sdk-ui-kit/api/sdk-ui-kit.api.md

```diff
@@ -1934,6 +1952,28 @@ export interface ITypographyProps {
     title?: string;
 }
 
+// @internal (undocumented)
+export interface IWorkspacePermissions {
```

Comment:
should not be interface for Permissions part of SPI? @no23reason @lupko

Reply:
You are correct, @ivan-nejezchleb, there is `IWorkspacePermissions` in sdk-backend-spi

Reply:
I used it in sdk-backend-spi instead of creating new thanks